CMakeLists.txt

```diff
@@ -3,9 +3,9 @@ cmake_minimum_required(VERSION 3.0)
 
 project(hpstr)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
I think it might be safer to keep this at 14 in master.

Reply:
Agreed! Sorry I did not mean to put this in here.  Will revert